src/taco-dependency-installer/installers/androidSdkInstaller.ts

```diff
@@ -122,31 +122,69 @@ class AndroidSdkInstaller extends InstallerBase {
 
         // Initialize values
         var androidHomeValue: string = path.join(this.installDestination, "android-sdk-macosx");
-        var addToPathTools: string = "$" + AndroidSdkInstaller.AndroidHomeName + "/tools/";
-        var addToPathPlatformTools: string = "$" + AndroidSdkInstaller.AndroidHomeName + "/platform-tools/";
-        var newPath: string = "\"$PATH:" + addToPathTools + ":" + addToPathPlatformTools + "\"";
-        var appendToBashProfile: string = "\n# Android SDK\nexport ANDROID_HOME=" + androidHomeValue + "\nexport PATH=" + newPath;
+        var fullPathTools: string = path.join(androidHomeValue, "tools/");
+        var fullPathPlatformTools: string = path.join(androidHomeValue, "platform-tools/");
+        var shortPathTools: string = "$" + AndroidSdkInstaller.AndroidHomeName + "/tools/";
```

Comment:
I updated the expandEnvVariables helper method to look for a ~ at the start of the path and expand it